test/xpu_api/random/conformance_tests/common_for_conformance_tests.hpp

```diff
@@ -53,14 +53,13 @@ typename Engine::scalar_type test(sycl::queue& queue) {
                     res.store(idx.get_linear_id(), dpstd_acc);
                 });
             });
+            queue.wait_and_throw();
```

Comment:
Could you please clarify why it was moved here?

Reply:
originally it was after catch() and it made no sense as we missed possible exceptions. As I understand we need first to wait till the end of execution and then catch all exceptions.